packages/ui/src/components/WithApolloClient.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
I think this functionality would be best coming from `@apollo/react-hoc` - unless I've missed the point somewhere?